server/server.go

```diff
@@ -136,7 +136,7 @@ func (server *Server) initDB() error {
 }
 
 func (server *Server) connectDB() error {
-	if err := migration.Init(); err != nil {
+	if err := migration.EnsureVersion(); err != nil {
```

Comment:
it should check no_init flag and reject running only when no_init = true and there are migrations pending